File: week3 - Application Development 1 CRUD with Cassandra/crud-java/src/main/java/com/datastax/week3/SelectRows.java
package com.datastax.week3;
import com.datastax.dse.driver.api.core.DseSession;
import com.datastax.oss.driver.api.core.cql.*;

public class SelectRows {
    
    public static void main(String args[]) {
       try (DseSession session = DseSession.builder().withCloudSecureConnectBundle(DBConnection.getConnectionPath())
           .withAuthCredentials(DBConnection.getUsername(), DBConnection.getPassword())
           .build()) {

            ResultSet results = session.execute(
                SimpleStatement.builder( "SELECT * FROM killrvideo.user_credentials WHERE email = ?")
                        .addPositionalValues("cv@datastax.com")
                        .build());
            Row row = results.one();
            System.out.println("***************************************************************************************");
            if (row == null) {
                System.out.println("No row selected");
            } else {
                System.out.format("%s %s %s\n",
                    row.getString("email"),
                    row.getString("password"),
                    row.getUuid("userid"));
            }
            System.out.println("***************************************************************************************");
        }
         catch(Throwable t) {
            System.out.println("Failed Select");
            t.printStackTrace(); 
        }
   }
}

File: week3 - Application Development 1 CRUD with Cassandra/crud-nodejs/select_rows.js
const connection = require('./db_connection')

// this is a select statement in nodejs
connection.client
.execute('SELECT * FROM killrvideo.user_credentials WHERE email = ?',
['cv@datastax.com'])
.then(function(result){
    result.rows.forEach(row => {
        console.log(row)
    })
    connection.client.shutdown()
})
.catch(function(error){
    console.log(error.message)
    connection.client.shutdown()
});